pkg/factory/risingwave_object_factory.go

```diff
@@ -1564,11 +1573,26 @@ func (f *RisingWaveObjectFactory) portsForStandaloneContainer() []corev1.Contain
 
 func (f *RisingWaveObjectFactory) setupStandaloneContainer(container *corev1.Container) {
 	container.Name = "standalone"
-	container.Command = []string{
-		risingwaveExecutablePath,
-		"standalone",
+
+	var imageVersion = utils.GetVersionFromImage(f.risingwave.Spec.Image)
+	if strings.HasPrefix(imageVersion, "v1.3") ||
```

Comment:
There is a potential risk: Some users may be using a special version provided by our engineers, which may have random names... 🤣 Don't know how to deal with this. @arkbriar  Do you have any ideas?

Reply:
Preferably there is a field in the CRD that helps enforce the feature. Such as,

```yaml
apiVersion: risingwavelabs.risingwave.com/v1alpha1
kind: RisingWave
spec:
  standaloneMode: v1 # value v0, v1, auto, defaults to auto, v0 means `standalone`, v1 means `single-node`, auto means auto detection.
```

Reply:
Done. Please take a look.